src/components/AudioAnalysis/SpAnalysisChordButtons.tsx

```diff
@@ -0,0 +1,116 @@
+import React from 'react';
+import {
+  makeStyles,
+  Typography,
+  Grid,
+  Select,
+  MenuItem,
+  InputLabel,
+  FormControl,
+} from '@material-ui/core';
+
+const useStyles = makeStyles(() => ({
+  select: {
+    color: '#ffffff',
+    width: '90%',
+    '&:before': {
+      borderColor: '#ffffff',
+    },
+    '&:after': {
+      borderColor: '#ffffff',
+    },
+  },
+  icon: {
+    fill: '#ffffff',
+  },
+}));
+
+const chords: { [key: string]: any } = {
```

Comment:
When selecting a major chord for example, there is no way to deselect it 🙂 maybe add an option "all"

Reply:
Done